tests/operators/test_agnostic_load_file.py

```diff
@@ -128,9 +128,9 @@ def test_aql_replace_existing_table(sample_dag, test_table, sql_server):
 )
 @pytest.mark.parametrize(
     "test_table",
-    [{"is_temp": True}, {"is_temp": False}],
+    [{}],
```

Comment:
What are we parameterizing over?

Reply:
So the table fixture returns the Table object(s) based on the config passed in the List and the table no. of config passed is the number of tables returned.

Maybe we can have a count instead of this weird-looking list if the tables are the default tables?

Reply:
empty list is weird, anything more explicit than this will be better

Reply:
@kaxil, I'll take care of this issue in a separate PR @tatiana also wanted a change in this Fixture. I'll address them both there.